frontend/src/common/constansts/footer.ts

```diff
@@ -0,0 +1 @@
+export const FOOTER_CONTENT = `© ${new Date().getFullYear()} Qpay. All rights are reserved`;
```

Comment:
```suggestion
export const FOOTER_CONTENT = `© ${new Date().getFullYear()} ${Titles.APP_NAME}. All rights are reserved`;
```

P.S I think Titles.APP_NAME should be "QPay" not "Qpay"